src/libYARP_OS/include/yarp/os/Log.h

```diff
@@ -70,7 +70,7 @@ class YARP_OS_API Log
     void info(const char *msg, ...) const YARP_ATTRIBUTE_FORMAT(printf, 2, 3);
     void warning(const char *msg, ...) const YARP_ATTRIBUTE_FORMAT(printf, 2, 3);
     void error(const char *msg, ...) const YARP_ATTRIBUTE_FORMAT(printf, 2, 3);
-    void fatal(const char *msg, ...) const YARP_ATTRIBUTE_FORMAT(printf, 2, 3);
+    void fatal YARP_NORETURN (const char *msg, ...) const YARP_ATTRIBUTE_FORMAT(printf, 2, 3);
```

Comment:
Good question, I wasn't even aware that you could add attributes there, but apparently it is the only place where all the compilers like it